src/FIO/Action.h

```diff
@@ -0,0 +1,113 @@
+// Copyright © 2017-2020 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "EOS/Name.h" // Name is reused
+#include "../Data.h"
+#include "../BinaryCoding.h"
+
+#include <string>
+#include <vector>
+
+namespace TW::FIO {
+
+/// Represents an authorization record (actor/permission pair)
+class Authorization {
```

Comment:
I think this can be moved to EOS namespace, same permission system

Reply:
Not 100%, leaving independent.